src/SDKs/CognitiveServices/dataPlane/Search/BingCustomSearch/BingCustomSearch/Properties/AssemblyInfo.cs

```diff
@@ -7,7 +7,7 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services Custom Search Client Library")]
 [assembly: AssemblyDescription("Provides API functions for consuming the Microsoft Azure Cognitive Services Custom Search API.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```

Comment:
Please do not bump the `AssemblyVersion` here however the `AssemblyFileVersion` must be updated to `1.2.0`